### PR TITLE
Update obligationsandreturns.html.md

### PR DESCRIPTION
**Pull request by mashukreza**
ginas updates from word doc

### CODE REVIEW

File: source/documentation/obligationsandreturns.html.md [hunk 1]
```diff
@@ -201,93 +295,44 @@ The [Loss claims](https://developer.service.hmrc.gov.uk/api-documentation/docs/a
...
-* capital gains
-* income from employment
-* any income from partnerships
-* any income from trusts
+Software should prompt customers to make sure they have considered the following potential additional income sources (links to the APIs where the functionality has been provided, we will continue to release additional functionality and will update this document when relevant) 
 
-The following are available:
+* Any income from bank/building society interest (supported in live)
+* Any income from dividends (supported in live)
+*	Any gift aid contributions they have made (supported in live)
```
Comment (james-work-account) [inline source/documentation/obligationsandreturns.html.md]: Same as above comment re indentation

File: source/documentation/obligationsandreturns.html.md [hunk 2]
```diff
@@ -201,93 +295,44 @@ The [Loss claims](https://developer.service.hmrc.gov.uk/api-documentation/docs/a
...
+NEED TO ADD INFO HERE ABOUT THE TAX CALC FOR CRYSTALLISATION AS MENTIONED ABOVE IN TAX CALC SECTION 
 
 The Calculation output provides a summary of each income source (for example self-employment, UK Property, UK bank and building society interest), plus a breakdown of allowances and reliefs applied, and a breakdown of the income tax and NIC payable.) broadly the equivalent of the current SA302.
 
-Once software has called the intent to crystallise endpoint, this will trigger a final liability calculation and software will receive a Calculation ID. 
+Once software has called the intent to crystallise endpoint, this will trigger a final liability calculation and software will receive a Calculation ID.
 
-The software must retrieve the intent to crystallise calculationId and display the calculation to the customer.  The customer must review this information and confirm it is complete and correct by sending the declaration.
+The software must us the  intent to crystallise calculationId to retrieve the final calculation and display that calculation to the customer. The customer must review this calculation  and confirm it is complete and correct by sending the declaration.
```
Comment (james-work-account) [inline source/documentation/obligationsandreturns.html.md]: "The software must _us_ the  intent to crystallise" -> "The software must _use_ the intent to crystallise"

File: source/documentation/obligationsandreturns.html.md [hunk 3]
```diff
@@ -201,93 +295,44 @@ The [Loss claims](https://developer.service.hmrc.gov.uk/api-documentation/docs/a
...
 
-If the customer thinks the calculation is incorrect as a result of incorrect data, they can go back and change any incorrect information they have provided. Once they have done this the software will have to call the ‘intent to crystallise’ API again to generate a new final liability.
+If the customer thinks the calculation is incorrect as a result of data they have submitted, they can go back and change the information, by resubmitting the relevant update with the correct information. . Once they have done this the software will have to call the ‘intent to crystallise’ API again to generate a new final liability.
```
Comment (james-work-account) [inline source/documentation/obligationsandreturns.html.md]: "update with the correct information. . Once they" - rogue full stop

File: source/documentation/obligationsandreturns.html.md [hunk 4]
```diff
@@ -305,52 +350,52 @@ Software must send the calculationId that matches the calculation the customer i
...
+A customer may want to retrieve previously submitted data, for example before making a change the customer may want to request the last update provided before sending in any changes. If the customer has recently started using your software, you may need to retrieve previous data.
 
 ### Periodic updates
 
-Software can use the [self employment or property periodic updates](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-api/2.0#selfemployment-business) endpoint to retrieve the list of updates made for that income source, or to find one or more period IDs.
-The period ID is then used with the ‘get a self employment/property (FHL or Non FHL) periodic update’ endpoint to retrieve data for that update.
+Software can use the list all self employment or property update periods endpoints to retrieve the list of updates made for that income source, or to find one or more period IDs. The period ID is then used with the ‘get a selfemployment/property (FHL or Non FHL) periodic update’ endpoint to retrieve data for that update.
```
Comment (james-work-account) [inline source/documentation/obligationsandreturns.html.md]: Space/hyphen in "selfemployment"?

File: source/documentation/obligationsandreturns.html.md [hunk 5]
```diff
@@ -305,52 +350,52 @@ Software must send the calculationId that matches the calculation the customer i
...
 
-There are some key points for changing previously submitted updates:
+Note: 
+
+* any changes that are made before the customer has crystallised is not a formal amendment For all changes to annual summary updates including Self-employment, FHL property business and non-FHL property business:
+
+## There are some key points for changing previously submitted updates:
+
+*	changes to Periodic updates - the update period you are trying to change must match the original update period exactly or it will be rejected
+*	changes to Annual updates - all figures previously supplied must be provided again, a Zero or a Null will overwrite any previously submitted information.
```
Comment (james-work-account) [inline source/documentation/obligationsandreturns.html.md]: "Zero" and "Null" are capitalised here but not on line 373

File: source/documentation/obligationsandreturns.html.md [hunk 6]
```diff
@@ -305,52 +350,52 @@ Software must send the calculationId that matches the calculation the customer i
...
 
-* changes to Periodic updates - the update period you are trying to change must match the original update period exactly or it will be rejected
-* changes to Annual updates - all figures previously supplied must be provided again, a Zero or a Null will overwrite any previously submitted information.
 
 ## Make an amendment after crystallisation
 
-If a customer wants to make any changes following crystallisation they have 12 months from the statutory filing date to do this (the statutory filing date is 31 January following the end of the tax year, or 3 months from receipt of a Notice to File by the taxpayer whichever is the later). 
+If a customer wants to make any changes following crystallisation they have 12 months from the statutory filing date to do this (the statutory filing date is 31 January following the end of the tax year, or 3 months from receipt of a Notice to File by the taxpayer whichever is the later).
```
Comment (james-work-account) [inline source/documentation/obligationsandreturns.html.md]: Is taxpayer one or two words? (same comment as before)

File: source/documentation/obligationsandreturns.html.md [hunk 7]
```diff
@@ -10,42 +10,59 @@ weight: 3
...
+*	For property businesses, an update for either FHL or Non FHL for the full quarter will mark the obligation as fulfilled, even if they have both income types. 
 
-Software will need to use that Calculation ID to use the [retrieve a tax calculation](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-api/2.0#tax-calculations_retrieve-a-tax-calculation_get_accordion) endpoint.
-Software should use that Calculation ID to call the [individual calculation](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individual-calculations-api/1.0#self-assessment_trigger-a-self-assessment-tax-calculation-test-only_post_accordion) endpoint a tax calculation API endpoint to get the result of the calculation.
+Software will need to use the following endpoints for each relevant source of income. For quarterly  updates:
+
+*	create a self-employment update period - this creates the update period and enables software to provide the summary totals of income and expenses for that specific self-employment business (this could be as little as a day or the whole 3 month obligation period)
```
[the remark this answers is not in the document]
Comment (mashukreza) [reply source/documentation/obligationsandreturns.html.md]: there will be hyperlinks - but I need to 'see' it on the portal; so these will be added on a second iteration

File: source/documentation/obligationsandreturns.html.md [hunk 8]
```diff
@@ -58,27 +75,86 @@ Note: In some cases, the obligation can take up to an hour to be confirmed as me
...
+*	Software can then call the relevant Get obligations APIs to establish whether or not the customer’s obligation has been met. 
+
+Note: 
+
+This step in the process is being replaced by a new Individual Calculation API – this new API will be going live in December 
+This step will be withdrawn on XX MONTH XXXX
+
+## Individual Calculation API
+
+* Software should use that Calculation ID to call the individual calculation endpoint a tax calculation API endpoint to get the result of the calculation.
```
[the remark this answers is not in the document]
Comment (mashukreza) [reply source/documentation/obligationsandreturns.html.md]: there has been more than one person working on this document - so there will be a few of these errors - thats expected

File: source/documentation/obligationsandreturns.html.md (hunk 3 above)
Comment (mashukreza) [reply source/documentation/obligationsandreturns.html.md]: this is draft so we are getting the content onto QA - these errrors are expected and will be removed on future iterations

File: source/documentation/obligationsandreturns.html.md (hunk 5 above)
Comment (mashukreza) [reply source/documentation/obligationsandreturns.html.md]: good spot - these should not be capitialised

File: source/documentation/obligationsandreturns.html.md (hunk 6 above)
Comment (mashukreza) [reply source/documentation/obligationsandreturns.html.md]: good call - it's one word (but two if its a council tax payer)

File: source/documentation/obligationsandreturns.html.md (hunk 4 above)
Comment (mashukreza) [reply source/documentation/obligationsandreturns.html.md]: minor typos like this are expected at this stage

File: source/documentation/obligationsandreturns.html.md [hunk 9]
```diff
@@ -121,21 +197,24 @@ If calculation errors are present, these errors can be returned to the customer
...
-Note: Data received must cover the whole accounting period.
+The process will take into account all the periodic and annual data already provided by the customer throughout the year.
 
 The customer must make sure they are happy with the information they have provided and add any additional information they have. This is likely to include tax and accounting adjustments, allowances or reliefs.
 
-Software should present to the customer the information listed below. This can either be totalled up by software, but we have provided APIs for you to get it from HMRC systems as well Business Income Source Summary (BISS) for self-employment or property:
+Note: 
+
+*	Data received must cover the whole accounting period.
+* We will be providing an API for customers to provide annual adjustments
```
[the remark this answers is not in the document]
Comment (mashukreza) [reply source/documentation/obligationsandreturns.html.md]: Not sure about this, should be neatly aligned right - but we have learnt that unexpected indentation/formatting can cause the pipeline to break (previously)

**Pull request merged**